Adding known issue for configuration policy (#3349)
open-cluster-management/rhacm-docs: release_notes/known_issues.adoc

When you install the gatekeeper operator on {ocp} version 4.9, the installation fails. Before you upgrade {ocp-short} to version 4.9.0., you must upgrade the gatekeeper operator to version 0.2.0. See link:../governance/create_gatekeeper.adoc#upgrading-gatekeeper-gatekeeper-operator[Upgrading gatekeeper and the gatekeeper operator] for more information.

[#config-policy-stuck]
=== Configuration policy listed complaint when namespace is stuck in _Terminating_ state
//2.5:20715

> When you have a configuration policy that is configured with `mustnothave` for the `complianceType` parameter and `enforce` for the `remediationAction` parameter, the policy is listed as compliant. If the policy is configured to delete a Kubernetes object, it does not verify if the object is deleted. Therefore, the Kubernetes object can be stuck in a `Terminating` state while the policy is listed as compliant.

Reviewer: ```suggestion
When you have a configuration policy that is configured with `mustnothave` for the `complianceType` parameter and `enforce` for the `remediationAction` parameter, the policy is listed as compliant after a deletion request is made to the Kubernetes API. Therefore, the Kubernetes object can be stuck in a `Terminating` state while the policy is listed as compliant.
```
